packages/wonder-blocks-popover/src/components/__tests__/popover.test.tsx

```diff
@@ -5,7 +5,7 @@ import {userEvent, PointerEventsCheckLevel} from "@testing-library/user-event";
 import {View} from "@khanacademy/wonder-blocks-core";
 import Button from "@khanacademy/wonder-blocks-button";
 
-import {fireEvent} from "@storybook/test";
+import {fireEvent} from "storybook/test";
```

Comment:
I'm not sure why this is here 😅... maybe it was meant to be RTL's fireEvent? (unrelated to your PR)

Reply:
Yeah, this was wrong. I changed it to be `fireEvent` from RTL and things work!